media/media/20190402-174047-4354479888.html

```diff
@@ -0,0 +1,122 @@
+<html>
```

Comment:
I'm not clear on what this file should do and how different it is from media/media/20190402-175447-4585677096.html

Reply:
media/media/20190402-175447-4585677096.html This file was autogenerated when sending out emails to users trying to signup. Its purpose was to give me a preview on the browser on how the email will look like in users inbox to aid me to style the emails. Regardless it should be added to the .gitignore as it in no way affect the code.

Reply:
Based on the above explanation, I think it's best if we remove this file from the PR